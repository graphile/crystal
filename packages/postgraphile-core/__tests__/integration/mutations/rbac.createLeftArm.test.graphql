#> rbac: true
#> options: { ignoreRBAC: false }

mutation {
  createLeftArm(input:{
    leftArm:{
      lengthInMetres: 0.66
    }
  }) {
    leftArm {
      nodeId
      id
      personId
      lengthInMetres
      mood
    }
  }
}
