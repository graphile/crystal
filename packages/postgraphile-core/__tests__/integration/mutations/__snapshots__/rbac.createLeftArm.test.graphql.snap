// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`execute 1`] = `
Object {
  "data": Object {
    "createLeftArm": Object {
      "leftArm": Object {
        "id": 100,
        "lengthInMetres": 0.66,
        "mood": "neutral",
        "nodeId": "WyJsZWZ0X2FybXMiLDEwMF0=",
        "personId": 3,
      },
    },
  },
}
`;
